pkg/controller.v1/tensorflow/pod_test.go

```diff
@@ -21,6 +21,7 @@ import (
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gstruct"
```

Comment:
```suggestion
```

Could you remove this dependency?

Reply:
Oh, you introduced the `IgnoreExtras`.
NVM

Reply:
I needed it in order to use `MatchFields`, `IgnoreExtras`, `Fields`.  For some reason `ContainElements(BeComparableTo` was not complaining that the fields did not match even though they appeared to be matching. Open to any other suggestions here.

Reply:
Uhm, It sounds curious.
Let me try to investigate it.
Anyway, I think that we can merge this for now.

Reply:
Thank you!